### DISCUSSION TITLE
podman packages 3.0.1 broken on ubuntu 20.04?

### DISCUSSION THREAD

**Discussion by aksdb** [containers/podman#9841] [Q&A]
Either the packages are broken or I totally miss something in the docs. I freshly installed podman (from `...:/kubic:/libcontainers:/stable/xUbuntu_20.04  podman 100:3.0.1-2`) but trying to do anything with it ends with:

```
WARN[0000] Error validating CNI config file /etc/cni/net.d/87-podman-bridge.conflist: [failed to find plugin "bridge" in path [/usr/libexec/cni /usr/lib/cni /usr/local/lib/cni /opt/cni/bin] failed to find plugin "portmap" in path [/usr/libexec/cni /usr/lib/cni /usr/local/lib/cni /opt/cni/bin] failed to find plugin "firewall" in path [/usr/libexec/cni /usr/lib/cni /usr/local/lib/cni /opt/cni/bin] failed to find plugin "tuning" in path [/usr/libexec/cni /usr/lib/cni /usr/local/lib/cni /opt/cni/bin]] 
```

The files are actually no where. And none of the packages contain it:

```
# dpkg-query -L podman
/.
/etc
/etc/cni
/etc/cni/net.d
/etc/cni/net.d/87-podman-bridge.conflist
/usr
/usr/bin
/usr/bin/podman
/usr/bin/podman-remote
/usr/lib
/usr/lib/systemd
/usr/lib/systemd/...
/usr/lib/tmpfiles.d
/usr/lib/tmpfiles.d/podman.conf
/usr/share
/usr/share/...
```

```
# dpkg-query -L podman-plugins
/.
/usr
/usr/libexec
/usr/libexec/cni
/usr/libexec/cni/dnsname
/usr/share
/usr/share/doc
/usr/share/doc/podman-plugins
/usr/share/doc/podman-plugins/changelog.Debian.gz
/usr/share/doc/podman-plugins/copyright
```

Is this something I should open a bug report for or am I just blind to something obvious?

**Comment by vrothberg** [comment]
@lsm5 PTAL

**Reply by aksdb** [reply]
Now that you bring my question up ... yesterday evening I managed to get it working by finding that there was another package missing: `containernetworking-plugins`. After installing this, the plugins are there.
I assume since the included config referenced them, they should be a required dependency, shouldn't they?

**Reply by afbjorklund** [reply]
Networking is optional (you can run with --network=none), so it is a "recommend" rather than a "depend"
